package.json

```diff
@@ -19,6 +19,14 @@
 	},
 	"homepage": "https://github.com/rysolv/markdown_ssg#readme",
 	"dependencies": {
-		"marked": "^4.0.12"
+		"fs-extra": "^10.1.0",
+		"i": "^0.3.7",
+		"imagemin": "^7.0.1",
+		"imagemin-jpegoptim": "^7.0.0",
+		"imagemin-pngquant": "^9.0.2",
+		"imagemin-webp": "^6.0.0",
+		"marked": "^4.0.12",
+		"npm": "^8.8.0",
+		"yargs": "^17.4.1"
```

Comment:
What have you done to my beautiful 1 dependency package.json 😢

Reply:
1. Do we need `npm` as a dependency? 
2. I think we can use a shell script instead of the build.js and avoid `yargs` 
3. I think the `i` package came from a typo. Example: `npm i i yargs`
4. `fs-extra` isn't necessary for just the copy function

Reply:
> What have you done to my beautiful 1 dependency package.json 😢

![image](https://user-images.githubusercontent.com/32417658/166151854-2ce86743-db67-40a3-9eb5-ca44541e36d7.png)

Reply:
> 1. Do we need `npm` as a dependency?
> 2. I think we can use a shell script instead of the build.js and avoid `yargs`
> 3. I think the `i` package came from a typo. Example: `npm i i yargs`
> 4. `fs-extra` isn't necessary for just the copy function

Yes i think a few of these are typos. With `fs-extra`, i was feeling lazy and just thought it was easier to install a new package instead of write a for loop 🤷

Will fix these